models/schemas/levelSchema.ts

```diff
@@ -240,7 +260,14 @@ LevelSchema.pre('save', function (next) {
     return next();
   }
 });
+LevelSchema.post('updateOne', async function(doc) {
```

Comment:
is the purpose of this so that we can call save on every level to force update the new calc field?